internal/compiler/tests/syntax/new_syntax/input_output2.slint

```diff
@@ -90,6 +105,13 @@ component Foo inherits Rectangle {
             input_model[42] += 12;
 //          ^error{Self assignment on a input property}
 
+            Glo.out1 = Glo.in1;
+//          ^error{Assignment on a output property}
+            Glo.inout1 = Glo.priv1;
+//                           ^error{'priv1' is private}
+            Glo.priv2 = Glo.out1;
+//              ^error{'priv2' is private}
+            Glo.in1 = 32;
```

Comment:
We could consider forbidding this and say that only the native code is supposed to modify global input properties

Reply:
Or we could issue a warning and say that global input doesn’t make sense and this will be interpreted as “in out”?